app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -39,7 +39,9 @@ def index
       respond_for_index :collection => @collection
     end
 
-    def index_response
+    # params :reload_host, Boolean, :default => false, :desc => "Host needs to be reloaded because of lazy accessor"
```

Comment:
This format of documentation is specific to our external API but `index_response` is internal to the controller. So I think we can remove this comment or go back to the old one :)

Here's how that style of comment affects the api docs:https://theforeman.org/plugins/katello/3.15/api/apidoc/v2/host_subscriptions.html